src/main/scala/com/bazaarvoice/sswf/model/history/HistoryFactory.scala

```diff
@@ -156,33 +157,39 @@ object HistoryFactory {
               currentStepStart = new DateTime(h.getEventTimestamp)
             }
             startedTimers.put(timerId, h)
-            None
+
+            val result = Success("Sleep started")
```

Comment:
this is wrong. in keeping with the activity task started events, the result should be "STARTED".